python/lsst/pipe/base/cmdLineTask.py

```diff
@@ -369,12 +373,16 @@ def __call__(self, args):
             self.log.MDC("LABEL", str([ref.dataId for ref in dataRef if hasattr(ref, "dataId")]))
         task = self.makeTask(args=args)
         result = None  # in case the task fails
+        exitStatus = 0 # exit status for the shell
         if self.doRaise:
             result = task.run(dataRef, **kwargs)
         else:
             try:
                 result = task.run(dataRef, **kwargs)
             except Exception as e:
+                exitStatus = 1          # n.b. The shell exit value is the number of dataRefs returning
+                                        # non-zero, so the actual value used here is lost
```

Comment:
Not sure what code standards say about trailing multiline comments, but I'm gonna guess they want two spaces between the end of the functional part and the start of the comment.